software/ompi/major-changes.php

```diff
@@ -9,6 +9,57 @@
 present an incompatibility vs. a prior version of Open MPI, or cause
 some other type of user-noticeable change in behavior.</em>
 
+<h2>v5.0.x series</h2>
```

Comment:
Why do we still have this page?  Future action item: this page should move into the RST docs.

Reply:
So should I just ommit everything here, and just link to v5.0.x news?

Reply:
It's probably ok to leave this for now; let's get 5.0.0 out the door.  We can clean up this (i.e., delete/redirect this page to an appropriate page in the RST docs) after the fact.  Sorry -- that's what I was trying to say as "Future action item: ...".